index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap">
+        <title>Portfolio</title>
+</head>
+
+<body>
+  
+  <header>
+    <nav title="navigation-bar">
+      <button class="nav-button1">
+        <span class="logo">Joy</span>
+      </button>
+  
+      <ul class="ul-nav" title="navigation-list">
+        <li><a class="navigationList" href="#portfolio">Portfolio</a></li>
+        <li><a class="navigationList" href="#about">About</a></li>
+        <li><a class="navigationList" href="#contactme">Contact</a></li>
+      </ul>
+    
+      <div class="nav-button2">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+
+<!--Page 1-->
+<section class="mobilemain" title="Joy's-Portfolio-Home-page">
+ <article class="headline">
+   <h1 id="heading">I’m Joy<br>
+     Glad to see you!</h1>
+   
+   <p id="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+   <div class="smedia" title="social-media-accounts">
+   
+   <h5 id="heading5">LET’S CONNECT</h5>
```

Comment:
- [ ] According to the design, this should be a button or a link. Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href. Please change it to a button/link. If you decide to make it a button, remember to add the button styling: Active, Hover and Pressed (see design)